code_generation/class.cpp

```diff
@@ -255,6 +255,15 @@ Typedef::List Class::typedefs() const
 void Class::addEnum(const Enum &enumValue)
```

Comment:
libkode is supposed to be a generic C++ code generator, at least so I thought.
If some code wants to add a enum to a class, it might not necessarily want to add special parsing/serialization methods.
IMHO this belongs in the layer above. Or at least it should be optional (e.g. a separate method to request the generation of those methods; cleaner than a parameter).

Reply:
Makes sense, then I think I will move it to the kode's printer itself.

Reply:
Hmm hmm, I consider printer to be at the same level :)

Anyone who wants to generate some C++ code would use Class and then Printer...

Doesn't this belong to the caller of addEnum() rather?

Reply:
Yes it makes sense. Now I rephrased the PR and moved away as much code as I could to the kode. (It is way better this way, so thanks for the hint).

I think the accessor adding to Enum should not be a trouble, however I would like to hear your opinion on the adding on an Invalid entry to each enum's declaration.